.github/workflows/extensions.yml

```
@@ -297,13 +297,23 @@ jobs:
       RELEASES: ${{ needs.fetch-releases.outputs.releases }}
 
     steps:
+      - name: Generate token for GitHub App
+        id: generate-token
+        uses: actions/create-github-app-token@v2
+        with:
+          app-id: 1129585
+          private-key: ${{ secrets.POSIT_CONNECT_PROJECTS_PEM }}
+          permissions: |
+            contents: write
+            pull-requests: write
+
       # Checkout main to commit the updated extension list
       # reduces the chance of conflicts when updating the extension list with
       # multiple running workflows
       - uses: actions/checkout@v4
```

nit, probably worth turning on dependabot for this kind of thing

```suggestion
      - uses: actions/checkout@v6
```